staging/src/k8s.io/cloud-provider/cloud.go

```diff
@@ -42,8 +42,8 @@ type ControllerClientBuilder interface {
 // Interface is an abstract, pluggable interface for cloud providers.
 type Interface interface {
 	// Initialize provides the cloud with a kubernetes client builder and may spawn goroutines
-	// to perform housekeeping activities within the cloud provider.
-	Initialize(clientBuilder ControllerClientBuilder)
+	// to perform housekeeping or run custom controllers specific to the cloud provider
```

Comment:
Document the semantics of the stop channel. (e.g. is it to tell the cloud provider to stop, or to find out when it stops?)

Reply:
done